core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -488,6 +488,31 @@ class ControllerApis(val requestChannel: RequestChannel,
     handleRaftRequest(request, response => new DescribeQuorumResponse(response.asInstanceOf[DescribeQuorumResponseData]))
   }
 
+  def handleElectLeaders(request: RequestChannel.Request): Unit = {
+    val electLeadersRequest = request.body[ElectLeadersRequest]
+
+    def sendErrorResponse(exception: Throwable): Unit = {
+      requestHelper.sendResponseMaybeThrottle(request, throttleMs => {
+        electLeadersRequest.getErrorResponse(throttleMs, exception)
+      })
+    }
+
+    if (!authHelper.authorize(request.context, ALTER, CLUSTER, CLUSTER_NAME)) {
+      sendErrorResponse(Errors.CLUSTER_AUTHORIZATION_FAILED.exception)
```

Comment:
Why not call `authHelper.authorizeClusterOperation(request, ALTER)` as other methods, It seems unnecessary to define a `sendErrorResponse` method.